mne_features/bivariate.py

```diff
@@ -0,0 +1,115 @@
+# Author: Jean-Baptiste Schiratti <jean.baptiste.schiratti@gmail.com>
+#         Alexandre Gramfort <alexandre.gramfort@inria.fr>
+# License: BSD 3 clause
+
+
+import numba as nb
+import numpy as np
+from math import sqrt
+from functools import partial
```

Comment:
wrong import order. first most core to periphery packages

Reply:
Fixed.